docs/PERFORMANCE.md

```diff
@@ -90,4 +90,4 @@ As a result, for a eventgen cluster with one server and one controller, we found
 
 In this architecture, the primary bottleneck is serializing and deserializing the data between processes.  We added the reduce step of the outputqueue primarily to handle modular input and file outputs where we needed a limited number of things touching a file or outputting to stdout.  Where we can parallelize this work, we can remove the majority of the CPU bottleneck of passing data around between processes.
 
-For this reason, in July of 2014, we added a config option to disable using the outputqueue.  In default/eventgen.conf, setting useOutputQueue to false or using `--disableOutputQueue` on the command line will output data directly from each GeneratorWorker rather than forcing the data back through the output queue.
\ No newline at end of file
+For this reason, in July of 2014, we added a config option to disable using the outputqueue. Setting useOutputQueue to false or using `--disableOutputQueue` on the command line will output data directly from each GeneratorWorker rather than forcing the data back through the output queue.
```

Comment:
This is wrong now, needs to change.  In eventgen 6 each plugin controls it's default style of output.  Either it can go in a queue for single threaded, or it'll run on the plugin itself.  I don't even know if this works anymore.